api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/PaymentCycleEndDate.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dhsc.htbhf.claimant.repository;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.util.UUID;
+
+/**
+ * Identifies a claim with a payment cycle due to expire on cycleEndDate.
+ */
+public interface PaymentCycleEndDate {
+
+    String getClaimIdString();
+
+    Timestamp getCycleEndDateTimestamp();
```

Comment:
Why a `Timestamp` and not a `LocalDateTime`? This differs from all the other date/time constructs we are using.

Reply:
This is the result of experimentation. Initially I had [UUID, LocalDate], but the native query returns a String and a Timestamp - hence the default methods to convert them to the desired values.

Reply:
I see, fine, the calling code should be using `getCycleEndDate` and not `getCycleEndDateTimestamp`, ok, that's fine.